db/migrate/20180120042303_create_expenses.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :expenses do |t|
       t.string :name, null: false
       t.date :due_date
-      t.string :frequency, null: false
+      t.integer :frequency, default: "weekly", null: false
```

Comment:
This will need to be an integer `0`

Reply:
Oh, of course, thanks for the pickup! Done in sha: 24fee5a